README.md

```diff
@@ -2,9 +2,51 @@
 
 Utility functions for keeping your code dry, when using General Update Pattern for d3.js
 
+#### Concepts
+
+This library uses two main concepts from d3.js good practises - General update pattern and data-selection join. 
+
+Joins are created using `join` function. It's simplest form it accepts node definition and data. It returns a function which accepts `selection` from d3. In this function will be evaluated data join, e.g. there will be created new selection and data will be bound to it.
```

Comment:
"the simpliest" or "simplier"?
consider using present simple instead of future one.